version2.res-examples.md

```diff
@@ -5,6 +5,7 @@
 * [Video/Audio](#videoaudio)
 * [With dealid](#with-dealid)
 * [With nurl and lurl](#with-nurl-and-lurl)
+* [With_burl](#with-burl)
```

Comment:
Since examples of notification URLs such as nurl and lurl are already included, it seems appropriate to add the burl examples there as well.